pom.xml

```diff
@@ -353,12 +353,12 @@
     </dependency>
     <dependency>
       <groupId>org.bouncycastle</groupId>
-      <artifactId>bcpkix-jdk15on</artifactId>
+      <artifactId>bcpkix-jdk18on</artifactId>
```

Comment:
Will this consider as BCR?

Reply:
Why BCR? The old one supports JDK5 and up, the new one supports JDK8 and up. We don't use or support below JDK8.

Reply:
If some one using JDK15 with JDBC version 3.13.33 (say last release version) and now after this fix we release JDBC 3.13.34 and if that customer try to use it, will it work? or he has keep using 3.13.33?

Reply:
The 15 is misleading. It is for JDK 1.5 / Java 5 not JDK 15. The minimum requirement for the jdbc driver is JDK 1.8